.changes/unreleased/FEATURES-20231031-160804.yaml

```
@@ -0,0 +1,6 @@
+kind: FEATURES
```

```suggestion
kind: ENHANCEMENTS
```

These changes are new fields to existing API requests and responses, so it's an enhancement from the perspective of the Go SDK library

---

[1dedefb](https://github.com/bastionzero/bastionzero-sdk-go/pull/42/commits/1dedefbf22aabacf48029e0eb75f17f06d6f4a4b)